### PR TITLE
doc: prepare new release 3.6.0

### PR DESCRIPTION
**Pull request by PhilippeAccorsi**
Ref #4150 
## Checkpoints

**For developers**

- [x] If relevant, manual tests have been done to ensure the stability of the whole application and that the involved feature works
- [x] The original issue is up to date w.r.t the latest discussions and contains a short summary of the implemented solution
- [x] Automated tests covering the feature or the fix, have been written, deemed irrelevant (give the reason), or an issue has been created to implement the test (give the link)
- [x] Make sure that:
  - if there are modifications in the Tracim configuration files (eg. `development.ini`), they are documented in `backend/doc/setting.md`
  - any migration process required for existing instances is documented
  - relevant people for these changes are notified
- [x] Original authors of the features included in a multi-feature branch (maintenance fixes -> develop, security fixes -> develop, …) should be part of the reviewers, especially if you encountered merge conflicts.

**For code reviewers**

- [x] The code is clear enough
- [x] If there are FIXMEs in the code, related issues are mentioned in the FIXME
- [x] If there are TODOs, NOTEs or HACKs in code, the date and the developer initials are present

**For testers**

- No manual test necessary here

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,43 @@
+# 3.6.0 / 2021-03-23
+
+### New Features
+
+- Advanced search available when search with Elasticsearch
```
Comment (Skylsmoi) [inline changelog.md:5]: Advanced search available with Elasticsearch

File: changelog.md [hunk 2]
```diff
@@ -1,3 +1,43 @@
...
+- Translation of Note and Comment directly in Tracim with Systran software (need a Systran account)
```
Comment (Skylsmoi) [inline changelog.md:6]: Translation of Note and Comment directly in Tracim using Systran software (require a Systran account)

File: changelog.md [hunk 3]
```diff
@@ -1,3 +1,43 @@
...
+[#4147](https://github.com/tracim/tracim/issues/4147),
+[#4149](https://github.com/tracim/tracim/issues/4149)
+- Agenda: [#3553](https://github.com/tracim/tracim/issues/3553)
+- Backend: [#2114](https://github.com/tracim/tracim/issues/2114)
+
+### Breaking Changes
+
+- :warning: You need to migrate your database before running this version. See the *Upgrading the Database to the Last Revision* section of the [migration documentation](backend/doc/migration.md) for more information  (issue #4133). We advise you to run this step after each upgrade of Tracim.
+If you use docker image, the migration is done automatically when new image is started.
+- ElasticSearch: Refactor of the indexing logic. It is necessary to drop the existing index and to create it again to use ElasticSearch, use the CLI command for this (issue #2660).
```
Comment (raphj) [inline changelog.md:27]:
```suggestion
- ElasticSearch: refactor of the indexing logic. It is necessary to drop the existing index and to create it again to use ElasticSearch, use the CLI command for this (issue #2660).
```

File: changelog.md [hunk 4]
```diff
@@ -1,3 +1,43 @@
...
+
+#### Backend configuration file (development.ini)
+
+- On existing Tracim installations using the docker image: It is necessary to update your development.ini (use this file [development.ini.sample](backend/development.ini.sample) to compare).
```
Comment (raphj) [inline changelog.md:31]:
```suggestion
- On existing Tracim installations using the docker image: it is necessary to update your development.ini (use this file [development.ini.sample](backend/development.ini.sample) to compare).
```

File: changelog.md [hunk 5]
```diff
@@ -1,3 +1,43 @@
...
+  - The `search.elasticsearch.index_alias` parameter has been renamed to `search.elasticsearch.index_alias__prefix`
+You need to delete, create and populate again your ElasticSearch index.
```
Comment (raphj) [inline changelog.md:33]: This is already said, no? Should we drop this sentence or a repetition does not hurt?

File: changelog.md (hunk 3 above)
Comment (Skylsmoi) [inline changelog.md:27]: the doc in the issue doesn't say to run `tracimcli index-populate` isn't it required?

File: changelog.md [hunk 6]
```diff
@@ -1,3 +1,43 @@
...
+
+### Other Changes
+
+- Sec: removing visibility of user email in Userschema used by the TLM
```
Comment (raphj) [inline changelog.md:37]:
```suggestion
- Security: email addresses of users are no longer returned by API when it is not strictly necessary.  
```

Comment (Skylsmoi) [inline changelog.md:37]: what is "Sec"?

File: changelog.md [hunk 7]
```diff
@@ -1,3 +1,43 @@
...
+- Translation service: new parameters available in the development.ini file to activate this feature and see the *Translation feature* section of the [setting documentation](backend/doc/setting.md) for more information (issue #4093).
```
Comment (raphj) [inline changelog.md:38]:
```suggestion
- Translation service: new parameters available in the development.ini file to activate this feature. See the *Translation feature* section of the [setting documentation](backend/doc/setting.md) for more information (issue #4093).
```

File: changelog.md [hunk 8]
```diff
@@ -25,7 +65,7 @@
 ### Breaking Changes
 
 - Some wordings in the user interface are changed (issue [#3901](https://github.com/tracim/tracim/issues/3901), [#4113](https://github.com/tracim/tracim/issues/4113), [#4114](https://github.com/tracim/tracim/issues/4114))
-- On existing Tracim installations using the docker image: It is necessary to update your development.ini (use this file [development.ini.sample](.backend/development.ini.sample) to compare). These three new parameters need to be added with this default path on your development.ini:
+- On existing Tracim installations using the docker image: It is necessary to update your development.ini (use this file [development.ini.sample](backend/development.ini.sample) to compare). These three new parameters need to be added with this default path on your development.ini:
```
Comment (raphj) [inline changelog.md:68]:
```suggestion
- On existing Tracim installations using the docker image: it is necessary to update your development.ini (use this file [development.ini.sample](backend/development.ini.sample) to compare). These three new parameters need to be added with this default path on your development.ini:
```

File: changelog.md (hunk 7 above)
Comment (Skylsmoi) [inline changelog.md:38]:
```suggestion
- Translation service: new parameters available in the development.ini file to activate this feature. See the *Translation feature* section of the [setting documentation](backend/doc/setting.md) for more information (issue #4093).
```

**Review by Skylsmoi — CHANGES_REQUESTED** [review]

File: changelog.md (hunk 3 above)
Comment (raphj) [reply changelog.md:27]: We could link to settings.md, Configure indexing and search to use Elasticsearch. But it is not very explicit. We would need a section explaining how to re-create the index. I can update settings.md accordingly if necessary.

Comment (PhilippeAccorsi) [reply changelog.md:27]: Right, I add information about where find command.

File: changelog.md (hunk 2 above)
Comment (raphj) [reply changelog.md:6]: requires

File: changelog.md (hunk 5 above)
Comment (PhilippeAccorsi) [reply changelog.md:33]: Thats true.

File: changelog.md (hunk 4 above)
Comment (PhilippeAccorsi) [reply changelog.md:31]: Done.

File: changelog.md (hunk 3 above)
Comment (PhilippeAccorsi) [reply changelog.md:27]: Done.

Comment (PhilippeAccorsi) [reply changelog.md:27]: Done.

File: changelog.md (hunk 2 above)
Comment (PhilippeAccorsi) [reply changelog.md:6]: Done

File: changelog.md (hunk 1 above)
Comment (PhilippeAccorsi) [reply changelog.md:5]: Done.

File: changelog.md (hunk 5 above)
Comment (PhilippeAccorsi) [reply changelog.md:33]: Done.

File: changelog.md (hunk 6 above)
Comment (PhilippeAccorsi) [reply changelog.md:37]: Done.

Comment (PhilippeAccorsi) [reply changelog.md:37]: Changed by security.

File: changelog.md (hunk 7 above)
Comment (PhilippeAccorsi) [reply changelog.md:38]: Done.

Comment (PhilippeAccorsi) [reply changelog.md:38]: Same comment. Done.

File: changelog.md (hunk 8 above)
Comment (PhilippeAccorsi) [reply changelog.md:68]: Done

**Comment by PhilippeAccorsi** [comment]
I have added information about delete, create **and populate** index.

**Review by Skylsmoi — APPROVED** [review]

File: changelog.md (hunk 3 above)
Comment (Skylsmoi) [reply changelog.md]: I was also pointing out that the issue #2660 doesn't tell to do the index-populaire, isn't it required?

File: changelog.md (hunk 2 above)
Comment (Skylsmoi) [reply changelog.md]: it misses raphj fixes

**Review by raphj — APPROVED** [review]

**Review by Skylsmoi — APPROVED** [review]

**Pull request merged**